gpkit/tests/test_repo.py

```diff
@@ -0,0 +1,112 @@
+"Implements tests for all external repositories."
```

Comment:
another docstring where it might be good to link to the overall explanation e-mail you sent